Marlin/src/pins/stm32f1/pins_BTT_SKR_E3_DIP.h

```diff
@@ -207,19 +216,82 @@
      *                    EXP1
      */
 
-    #define BTN_ENC                         PB6
-    #define BTN_EN1                         PA9
-    #define BTN_EN2                         PA10
+    #define BTN_ENC                  EXP1_02_PIN
+    #define BTN_EN1                  EXP1_03_PIN
+    #define BTN_EN2                  EXP1_05_PIN
 
-    #define DOGLCD_CS                       PB8
-    #define DOGLCD_A0                       PB9
-    #define DOGLCD_SCK                      PA15
-    #define DOGLCD_MOSI                     PB7
+    #define DOGLCD_CS                EXP1_07_PIN
+    #define DOGLCD_A0                EXP1_06_PIN
+    #define DOGLCD_SCK               EXP1_01_PIN
+    #define DOGLCD_MOSI              EXP1_08_PIN
     #define FORCE_SOFT_SPI
     #define LCD_BACKLIGHT_PIN               -1
 
+  #elif ENABLED(FYSETC_MINI_12864_2_1)
+
+    #ifndef NO_CONTROLLER_CUSTOM_WIRING_WARNING
+      #error "CAUTION! ZONESTAR_LCD requires wiring modifications. See 'pins_BTT_SKR_MINI_E3_DIP.h' for details. (Define NO_CONTROLLER_CUSTOM_WIRING_WARNING to suppress this warning.)"
```

Comment:
Should probably change to `CAUTION! FYSETC_MINI_12864_2_1 requires wiring modifications.`

Reply:
oops copy and paste error..